pkg/cmd/kind/build/nodeimage/nodeimage.go

```diff
@@ -68,7 +68,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 	cmd.Flags().StringVar(
 		&flags.KubeRoot, "kube-root",
 		"",
-		"path to the Kubernetes source directory (if empty, the path is autodetected)",
+		"DEPRECATED: please switch to just the argument. Path to the Kubernetes source directory (if empty, the path is autodetected)",
```

Comment:
@aojea in the future `s/just//`

Reply:
it didn't read well to me, but I grepped and I saw in other place so I assumed that was not wrong :smile:

Reply:
we're also avoiding words that imply a presumption of [low] difficulty in docs etc. in Kubernetes now AIUI.